### PR TITLE
release notes for v0.19.0

### PR DESCRIPTION
**Pull request by loriab**
## Status

- [x] Code base linted
- [x] Ready to go

### CODE REVIEW

**Comment by loriab** [comment]
@dgasmith, the fields list grew quite a bit https://github.com/samuelcolvin/pydantic/blob/master/pydantic/fields.py#L254-L277, so could you check over the change here, please? `typing_map[12]` was the only error throwing in the test cases.

File: qcelemental/util/autodocs.py [hunk 1]
```diff
@@ -44,6 +44,12 @@ def parse_type_str(prop) -> str:
         fields.SHAPE_LIST: "List",
         fields.SHAPE_SINGLETON: "Union",
     }  # yapf: disable
+    try:
```
Comment (dgasmith) [inline qcelemental/util/autodocs.py:47]: Ah, I was waiting for the 1.8 regressions. I pin pydantic minor these days until 1.x.1 is out, its getting to be a pretty complex system and regardless of their expansive test suite, items filter through.

**Review by dgasmith — APPROVED** [review]
LGTM! Nice.

File: qcelemental/util/autodocs.py (hunk 1 above)
Comment (loriab) [reply qcelemental/util/autodocs.py:47]: the big v1.8 one was intended (see breaking: https://github.com/samuelcolvin/pydantic/releases/tag/v1.8). this little typing one was a surprise. Is it supposed to be `Dict` or `Dict[{}]`?

**Pull request merged**